core/controllers/ingest.py

```diff
@@ -72,8 +72,8 @@ def ingest(body: dict, excel_file: FileStorage) -> tuple[dict, int]:
 
     original_workbook = extract_data(excel_file=excel_file)
     try:
-        if ptv_schema := ingest_dependencies.pre_transformation_validation_schema:
-            pre_transformation_validations(original_workbook, ptv_schema, auth)
+        if ptv_schema := ingest_dependencies.initial_validation_schema:
```

Comment:
I'm not totally happy with the name either but find "pre transformation validation" a mouthful like you say.. Open to other suggestions.. Unless we differentiate by using the term "check" for this instead of "validation", but that is also a bit ambiguous